### ISSUE TITLE
Easy to Read License

### ISSUE THREAD

**Issue by nprail** [lukevella/Rallly#34]
I was trying to read Rallly's license, but I was having a really hard time doing so because the format is rather crazy. So I went and found the CC-BY-NC license from CC and read it instead. It is CC-BY-NC v3, correct? Can the license be replaced with a more readable formatted version? I believe this is the same license formatted better: https://creativecommons.org/licenses/by-nc/3.0/legalcode.txt

**Comment by nprail** [comment]
@lukevella I'm going to start working on some of these ideas. I think I'll start with adding an option to put a password on events. 

My co-worker from @Filiosoft will possibly help.

**Comment by thekyriarchy** [comment]
CC-BY-NC is not compatible with free software or open source. 

The current license text reads, 

> CREATIVE COMMONS PUBLIC LICENSE ("CCPL" OR "LICENSE")

...which is not an actual license. Not to mention Creative Commons licenses are not made for code, and do not protect code properly. 

This is a violation of the Creative Commons organization's trademarks, is incompatible with open source / free software, and is probably legally very hairy. You likely would want the AGPLv3: https://www.gnu.org/licenses/agpl-3.0.en.html

**Comment by thunfischbrot** [comment]
Maybe @lukevella intended to reference CC0?
https://wiki.creativecommons.org/wiki/CC0_FAQ#May_I_apply_CC0_to_computer_software.3F_If_so.2C_is_there_a_recommended_implementation.3F

**Comment by thekyriarchy** [comment]
No, the actual text is taken from the Creative Commons Attribution (CC-BY) 3.0 Unported license code. Not only has it been deprecated by version 4.0, but as that link points out, CC Licenses are not for software: 

> We recommend against using Creative Commons licenses for software. Instead, we strongly encourage you to use one of the very good software licenses which are already available. We recommend considering licenses made available by the Free Software Foundation

and 

> Unlike software-specific licenses, CC licenses do not contain specific terms about the distribution of source code, which is often important to ensuring the free reuse and modifiability of software. Many software licenses also address patent rights, which are important to software but may not be applicable to other copyrightable works. Additionally, our licenses are currently not compatible with the major software licenses, so it would be difficult to integrate CC-licensed work with other free software. Existing software licenses were designed specifically for use with software and offer a similar set of rights to the Creative Commons licenses.

https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

**Comment by lukevella** [comment]
Hi all. Thanks bringing this to my attention. I admin, the license was a bit of an after-thought. I used to use CC-BY-NC for design work but I see now that it doesn't work for source code.

I've went ahead and changed the license to MIT. I am not planning on doing any further development on this codebase so happy for anyone to use it any way they wish. And I'll  go ahead and close this issue.

Thanks again.
